### PR TITLE
Enhances for translation and contribution readmes

### PR DESCRIPTION
**Pull request by Vovkiv**
TLDR: https://github.com/Vovkiv/GoZen-ru-uk/blob/master/translations/README.md see changes here and here: https://github.com/Vovkiv/GoZen-ru-uk/blob/master/CONTRIBUTING.md

So, I did some general enhances to documentation for CONTRIBUTION.md and translations/README.md
For CONTRIBUTION:
* Added link to download Godot to save some clicks for contributors. In future, you might want to consider to provide link to exact version that contributor need to use using this archive: https://godotengine.org/download/archive/
*  Added link to Godot docs. Again, to save some clicks.
* Added link to translations/readme to point out that contributor can help with translations.

For translations/readme:
* Some general styling. Like, for example, using code styling instead of italic for better highlighting.
* Merged list of available translations with who contributed to specific language, since I don't see any reason why they should be separated. Also sorted this list in alphabetical order. Also fixed typos for Russian and Ukrainian.
* Removed "Future languages" section, since it doesn't provide any meaningful information.
* Removed "All languages welcome!" section, since it is also doesn't contribute with any meaningful information.
* Added link to list of all available language codes on Godot docs. Because there several standards for language codes, we need to make sure that contributor will use right one.
* Renamed "New language?" to "Adding new translation"; "Updating an already existing language" to "Updating an already existing translation"
* Added jumping to section for "Adding new translation" and "Updating an existing translation", so user can click on them.

Some ideas that should be implemented later:
* Add link to download page to exact version of Godot that contributor need to use.
* Add option for translation contributors to add some way to feedback in translation files or in README file, like email or personal site link. And make this information visible, so if end user will catch some mistranslated or typo, they can open some "help" menu, see email link to translator and write email to them "Hey, I found typo! Can you fix it?"
* Add section how to test translation, once it become possible to do so.

### CODE REVIEW

**Review by voylin — APPROVED** [review]

**Comment by voylin** [comment]
Thank you very much for these changes/additions! ^^
Very helpful as I am not that great at nicely explaining things haha

Approved and merged, thanks agian ;)
